cmd/kubeadm/app/util/endpoint.go

```diff
@@ -115,9 +115,9 @@ func ParseHostPort(hostport string) (string, string, error) {
 // ParsePort parses a string representing a TCP port.
 // If the string is not a valid representation of a TCP port, ParsePort returns an error.
 func ParsePort(port string) (int, error) {
-	portInt, err := strconv.Atoi(port)
-	if err == nil && (1 <= portInt && portInt <= 65535) {
-		return portInt, nil
+	portInt, err := netutils.ParsePort(port, false)
+	if err == nil {
+		return int(portInt), nil
```

Comment:
ParsePort() already returns int, no need for a cast, unless I am missing something?  I expect `go vet` will yell at you.